dts/bindings/pinctrl/st,stm32f1-pinctrl.yaml

```diff
@@ -0,0 +1,97 @@
+            With:
+            - port: The gpio port index (PA = 0, PB = 1, ..., PK = 11)
+            - line: The line offset within the port (PA0 = 0, PA1 = 1, ..., PA15 = 15)
+            - function: The configuration mode, can be:
+            * 0 : Alternate function output
+            * 1 : Input
+            * 2 : Analog
+            - remap: The pin remapping configuration. It allows to assign the pin
+            function to a different peripheral. Remain configuration can be:
+            * 0 : No remap
+            * 1 : Partial remap 1
+            * 2 : Partial remap 2
+            * 3 : Partial remap 3
+            * 4 : Full remap
+            To simplify the usage, macro is available to generate "pinmux" field.
+            This macro is available here:
+              -include/dt-bindings/pinctrl/stm32f1-pinctrl.h
+            Some examples of using macro:
+            /* GPIO A9 set as alernate with no remap */
+            ... {
+                     pinmux = <STM32F1_PINMUX('A', 9, ALTERNATE, REMAP_NO)>;
+            };
+            /* GPIO A9 set as alernate with full remap */
+            ... {
+                     pinmux = <STM32F1_PINMUX('A', 9, ALTERNATE, REMAP_FULL)>;
+            };
+            /* GPIO A9 set as input */
+            ... {
+                     pinmux = <STM32F1_PINMUX('A', 9, GPIO_IN, REMAP_NO)>;
+            };
+
+        bias-disable:
+          required: false
+          type: boolean
+          description: Pin bias (push-pull) disabled. This is the default pin
+            configuration and will be applied if no bias- property is
+            specified. Only available in input mode ("floating" configuration).
+
+        bias-pull-down:
+          required: false
+          type: boolean
+          description: Weak pull down resistor enabled. Not compatible with
+            output configuration modes (atlernate or general purpose output).
+
+        bias-pull-up:
+          required: false
+          type: boolean
+          description: Weak pull up resistor enabled. Not compatible with
+            output configuration modes (atlernate or general purpose output).
+
+        drive-push-pull:
+          required: false
+          type: boolean
+          description: Pin driven actively high and low. This is the default pin
+            configueration and will be applied if no drive- property is
+            specified. Only valid for output configuration modes.
+
+        drive-open-drain:
+          required: false
+          type: boolean
+          description: Pin driven in open drain. Only valid for output
+            configuration modes.
+
+        slew-rate:
+          required: false
+          type: string
+          default: "max-speed-10mhz"
+          enum:
+          - "max-speed-10mhz"       # Default
+          - "max-speed-2mhz"
+          - "max-speed-50mhz"
+          description: Pin output mode. Only applies to output mode (alternate).
```

Comment:
Description needs to be fixed here

Reply:
Any suggestion ? I tried to stay close from the refman.

Reply:
The description says `Pin output mode`, it should explain that GPIO signal slew-rate is configured so that the listed maximum speed can be achieved.

Reply:
fixed